tracepath.c

```diff
@@ -609,6 +609,10 @@ int main(int argc, char **argv)
 
  done:
 	freeaddrinfo(result);
+	if(ctl.pktbuf) {
+		free(ctl.pktbuf);
+		ctl.pktbuf = NULL;
```

Comment:
What is the reason to set `ctl.pktbuf` `NULL`? It's not being used later.
Also, there is another `exit()` via `error()`, which does not call `free(ctl.pktbuf)`:

https://github.com/iputils/iputils/blob/33e78be2e60ed9ac918dec13271d1bd9dce6e94e/tracepath.c#L570-L589

Reply:
no special reason，just used to it. feel free to delete "set ctl.pktbuf NULL".

about exit by errors but no free(ctl.pktbuf), i think we could change to "error(0, errno, "IPV6_UNICAST_HOPS");  free(ctl.pktbuf); exit(1);"

I will modify the PR later